test/sqllogictest/arrays.slt

```diff
@@ -65,3 +65,71 @@ true
 
 query error ANY comparison op <= not yet supported
 select 'hello'::text <= ANY(ARRAY['there'::text])
+
+# Duplicates of cockroach/array.slt. todo@jldlaughlin: Remove when we support that file.
```

Comment:
Is this the right way to be adding Cockroach tests temporarily? Or should I be unblocking parts of that file?